src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
+      fileNames.push(fileName);
+
+      apiV1FileObject[fileName] = {
+        path: fileObject[key].s3Path,
+        size: fileObject[key].size,
+        valid: fileObject[key].valid,
+        name: fileName,
+        upload: {
+          status: fileObject[key].uploadStatus,
+        },
+      };
+    });
+    return { apiV1FileObject, fileNames };
+  };
+
+  const sampleTechnologyConvert = (technology) => {
+    switch (technology) {
```

Comment:
For two cases wouldn't it be easier to use if else?

And if we have many cases in the future we could probably just store all possibilities in an object instead of using a switch